test/native/helpers.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { act, render, fireEvent } from '@testing-library/react-native';
+
+// Set up the mocks for getting the HTML output of the editor
+let triggerHtmlSerialization = () => {};
+let serializedHtml;
+const bridgeMock = jest.requireMock( '@wordpress/react-native-bridge' );
+bridgeMock.subscribeParentGetHtml = jest.fn( ( callback ) => {
+	triggerHtmlSerialization = callback;
+} );
+bridgeMock.provideToNative_Html = jest.fn( ( html ) => {
+	serializedHtml = html;
+} );
+
+export async function initializeEditor( { initialHtml } ) {
+	const Editor = require( '@wordpress/edit-post/src/editor' ).default;
```

Comment:
Is there an advantage from mixing the use of `import` and `require` in this file? If not, I think it'd be worthwhile to consistently utilize `import`.

Reply:
Not really, we could use `import`, however, I'd like to note that we would have to disable the lint rule that prevents WordPress dependencies to be imported with path access. The import will look like this:
```
// eslint-disable-next-line no-restricted-syntax
import Editor from '@wordpress/edit-post/src/editor';
```

Reply:
Ah, OK. I keep forgetting it is an internal module. 😄 I think either approach is fine. Maybe we should leave a comment explaining why we are importing an internal module? My perception is that it is the only way to set the `initialHtml`. Is that correct?

Reply:
>Ah, OK. I keep forgetting it is an internal module. 😄 I think either approach is fine.

Ok, I rather have it all as imports so I'll apply your suggestion.

>Maybe we should leave a comment explaining why we are importing an internal module?

Sure! I'll drop a comment in the code explaining the reason 👍 .

>My perception is that it is the only way to set the initialHtml. Is that correct?

Yeah, although it would be possible to initialize the editor as the main apps do by calling the top-level function [`doGutenbergNativeSetup`](https://github.com/WordPress/gutenberg/blob/7e360e43a50da560aa8ba30a2432464e520eb2ec/packages/react-native-editor/src/index.js#L147-L150). However, we would need to figure out a way to get the editor element to pass it to the `@testing-library/react-native` render function, my first approach would be to mock `AppRegistry.registerComponent`.

Here I describe the flow that explains how the Editor component is rendered:
1. `react-native-editor` initialization
    1. [`gutenbergSetup`](https://github.com/WordPress/gutenberg/blob/6da0390f864c68dc69494d50d586e8c0d841835e/packages/react-native-editor/src/index.js#L149)
    2. [`initializeEditor`](https://github.com/WordPress/gutenberg/blob/6da0390f864c68dc69494d50d586e8c0d841835e/packages/react-native-editor/src/index.js#L47-L48)
2. [Editor initialization](https://github.com/WordPress/gutenberg/blob/6da0390f864c68dc69494d50d586e8c0d841835e/packages/edit-post/src/index.native.js#L31)
3. [Editor render](https://github.com/WordPress/gutenberg/blob/6da0390f864c68dc69494d50d586e8c0d841835e/packages/element/src/react-platform.native.js#L17-L31)

Besides, initializing the editor this way would do the full setup including, registering all blocks, setup localizations, etc., that we might don't want for all tests.